addon/mixins/routes/application.js

```diff
@@ -31,6 +31,13 @@ export default Ember.Mixin.create({
         outlet: outlet,
         parentView: parentView
       });
+
+      // Check if another modal exists then give it focus
+      Ember.run.later(function() {
+        var remainingViewElement = Ember.$('[role="dialog"]:last-child').closest('.ember-view');
```

Comment:
Also, does this code throw an error when there are no more modals rendered? I would expect `remainingViewElement` to not have a `removeAttr` method if there are no other modals present. It might be worth throwing in an `if` statement as a safety net here.